.gitmodules

```diff
@@ -1,6 +1,8 @@
 [submodule "evm"]
 	path = evm
-	url = https://github.com/rust-blockchain/evm
+	url = https://github.com/aurora-engine/sputnikvm
+	branch = feat/london-support
```

Comment:
TODO: revert back to upstream after https://github.com/rust-blockchain/evm/pull/54 is merged